landkreise/get-heinsberg-kreis.py

```diff
@@ -0,0 +1,29 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+
+import scrape
+import helper
+
+main_url = "https://www.kreis-heinsberg.de/aktuelles/aktuelles/?pid=5142"
```

Comment:
this is actually only the news page from 26. March, it doesn't update.

There is a newer page available, see https://www.kreis-heinsberg.de/aktuelles/aktuelles/?pid=5146

So you would have to fetch the most current news page and extract the cases from there.

Reply:
at the moment they say "Heinsberg. Der aktuelle Stand der Corona-Infektion im Kreis Heinsberg (30. März, 12 Uhr)" so i thought they update their numbers there. But todays numbers are on https://www.kreis-heinsberg.de/aktuelles/aktuelles/?pid=5147. So i am not sure. it looks you are right.